source/help/policies/format_requirements.md

```diff
@@ -13,7 +13,7 @@ In addition there are a limited set of specific format requirements.
 - 10 to 14 point type
```

Comment:
Suggest adding "within the body of the work" as otherwise we may see people who complain or ask about other font sizes.